introduction.md

```diff
@@ -1 +1,5 @@
-Hello, this is introduction of Sergey Shihov to complete the codeschool course. I like JavaScript actually!
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?